### PR TITLE
feature: allow to hide tag history to simplify the user interface

### LINKED ISSUE #358: Feature request: Allow hiding the history feature to reduce complexity of the user interface
**Is your feature request related to a problem? Please describe.**
I would like to use docker-registry-ui to inform users about the available tags for our images. The users of this interface are not interested in the technical backgrounds how it works and I worry that it would confuse or overwhelm them.

Just from the eyes of a non-maintainer or a less container-familiar admin, I think it has too much in-depth info having the non-descriptive layers and thinking it is required to get the Dockerfile or something else.

**Describe the solution you'd like**
Therefore, I propose to add a new optional feature toggle "HIDE_TAG_HISTORY" to hide the tag history, similar to "SHOW_CONTENT_DIGEST".

**Describe alternatives you've considered**
I thought about reducing the information in the tag history view itself. But it would be more fitting to have a details view to see the information like Env, Volumes, Entrypoint, etc. just for the topmost, latest layer in a different view then history.

**Additional context**
I have prepared a commit that introduces this feature. Shall i just open the PR to discuss the diff?

I would also like to add the option to customize the title for the browser tab and in the toolbar. Would you be open for another PR that adds this?

Thanks anyway for the project. It is very simple to work with!

### PR DESCRIPTION
**Pull request by lukengda**
I would like to use docker-registry-ui to inform users about the available tags for our images. The users of this interface are not interested in the technical backgrounds how it works and I worry that it would confuse them.

Note: One needs to be aware, that this is data is available nonetheless. This is only a user interface switch for hiding the button.

If desired, i could also rephrase the variables to "show-tag-history", defaulting to true, to match the name of other options. 

Is there anything else that is required for the merge request?

Closes #358

### CODE REVIEW

**Comment by Joxit** [comment]
Hi, thank you for your contribution, I think you're right, you should rename the variable to `show-tag-history`

**Comment by lukengda** [comment]
I reversed the logic and called it `SHOW_TAG_HISTORY`, defaulting to `true`. Hope this fits better now :)

**Review by Joxit — APPROVED** [review]
LGTM, thanks for your contribution

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Some env options are available for use this interface for **only one server** (w
 - `CATALOG_MAX_BRANCHES`: Set the maximum repository/namespace to expand (e.g. `joxit/docker-registry-ui` `joxit/` is the repository/namespace). Can be 0 to disable branching. (see [#319](https://github.com/Joxit/docker-registry-ui/pull/319)). (default: `1`). Since 2.5.0
 - `TAGLIST_PAGE_SIZE`: Set the number of tags to display in one page. (default: `100`). Since 2.5.0
 - `REGISTRY_SECURED`: By default, the UI will check on every requests if your registry is secured or not (you will see `401` responses in your console). Set to `true` if your registry uses Basic Authentication and divide by two the number of call to your registry. (default `false`). Since 2.5.0
-
+- `SHOW_TAG_HISTORY`: Whether to show the tag history feature or not. Allows to simplify the user interface by hiding it form the tag list if set to `false`. (default: `true`).
```
Comment (Joxit) [inline README.md:122]: You text is a way to much subjective "Allows to simplify the user interface". I will change it before the next release.

**Pull request merged**